scripts/wsrep_sst_xtrabackup-v2.sh

```diff
@@ -843,6 +888,9 @@ cleanup_joiner()
         wsrep_log_debug "Cleaning up fifo file $progress"
         rm $progress
     fi
+    if [[ -n "${JOINER_SST_DIR}" ]]; then
```

Comment:
Need to think about this.  If an error occurred, we kept this directory around for debugging.

Reply:
For example,
            wsrep_log_error "******************* FATAL ERROR ********************** "
            wsrep_log_error "Move failed, keeping ${DATA} for further diagnosis" \
                            "Check ${DATA}/innobackup.move.log for details"
            wsrep_log_error "Line $LINENO"
            cat_file_to_stderr "${DATA}/innobackup.move.log" "ERR" "innobackup.move.log"
            wsrep_log_error "****************************************************** "